core/src/main/java/jenkins/PluginSubtypeMarker.java

```diff
@@ -54,7 +54,6 @@
  */
 @SupportedAnnotationTypes("*")
 @MetaInfServices(Processor.class)
-@SuppressWarnings("Since15")
```

Comment:
Whatever this is, it smells like "since Java 5". It seems like that's in the rear-view mirror now that we're on Java 8.